.github/workflows/workflow.yaml

```
@@ -0,0 +1,11 @@
+name: Build Test
+on: [push]
+
+jobs:
+  build_job:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Create Docker Image
+        uses: ./
```

We should keep Github specific files in one place and not spread them across the repo.

```suggestion
        uses: ./github/actions/build-from-dockerfile
```

This requires the `action.yaml` to be moved to `.github/actions/build-from-dockerfile/action.yaml`